cv_next/app/feed/components/feed.tsx

```diff
@@ -70,7 +106,17 @@ export default function Feed() {
       } else {
         setLoadMore(false);
       }
+      if(filters.categoryId){
+        params.delete("category");
+        console.log("params before", params);
```

Comment:
Delete the log. If you want to keep it, put a debug log here.

Reply:
ok